### ISSUE TITLE
no results in Components page when trying the component authoring

### ISSUE THREAD

**Issue by yas-globe** [AdobeDocs/experience-manager-learn.en#111]
Issue in ./help/getting-started-wknd-tutorial-develop/component-basics.md

Component Authoring
i tried this:
$ git clone git@github.com:adobe/aem-guides-wknd.git --branch tutorial/component-basics-start --single-branch
and this
$ git clone git@github.com:adobe/aem-guides-wknd.git

then mvn -PautoInstallSinglePackage clean install
build success.

there are components in /apps/wknd/components

**Comment by godanny86** [comment]
Hi @yas-globe ,

What version of AEM are you using? Do you have a screenshot of the page?

**Comment by yas-globe** [comment]
Thanks for replying. 
AEM build 2020.12.4676
This sounds stupid, but i stopped working on it since i posted this. But now it's working.
I was working on another tutorial instead. I did not touch the code whatsoever. The only thing i have done is I have turned off the server every night, since. Which I already did while troubleshooting before. The Components are now populated in the Side Panel, It's now working fine. It's like the author server had to sleep 8 hours for it to work hahaha
